types/get-caller-file/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for get-caller-file 1.0
+// Project: https://github.com/stefanpenner/get-caller-file#readme
+// Definitions by: Klaus Meinhardt <https://github.com/ajafff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function getCallerFile(position?: number): string;
+declare namespace getCallerFile {}
```

Comment:
This should no longer be necessary with `esModuleInterop`.

Reply:
Though I think it's still appropriate for your test file to continue using `import = require`

Reply:
@DanielRosenwasser Technically you are right, but that requires every user to either use `import = require` (which isn't possible with `module: es2015` IIRC) or use typescript@2.7 with `esModuleInterop` enabled (which still has some bugs in the current version)

Personally I don't like `import = require` and hold off on using esModuleInterop for a while.

Reply:
While I understand the desire to avoid `import = require`, it's just syntax that makes it more explicit as to what will happen at runtime. [It's been a guideline on DefinitelyTyped not to add empty namespaces for a while](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/6d29db7d85a38e62105272a0007fca891f494736#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)

Besides, if you want this behavior, we've had `allowSyntheticDefaultImports` since before 2.0.

Reply:
got it, thanks. I updated the PR to fix this